databases/week3/meal_sharing_queries.sql

```diff
@@ -0,0 +1,90 @@
+USE `meal-sharing`;
+
+-- Meal Queries -- 
+-- get all meals
+SELECT * FROM meal; 
+
+-- Add a new meal
+INSERT INTO `meal` (id, title, description, location, `when`, max_reservation, price, created_date) VALUES (6, 'Homestyle Vietnamese food', 'Crispy spring roll serve with sweet chili sauce', 'Christainshavn', '2023-02-01 10:00', 2, 45.50, '2023-01-31');
+
+-- Get a meal with any id, fx 1
+SELECT * FROM meal WHERE id = 1;
+
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE meal SET max_reservation = 3 WHERE id = 6; 
+
+-- Delete a meal with any id, fx 1
+DELETE FROM meal WHERE id = 6; 
+
+-- Reservation Queries -- 
+-- Get all reservations
+SELECT * FROM reservation;
+ 
+-- Add a new reservation
+INSERT INTO `reservation` (id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES(8, 1, 4, '2023-02-09', NULL , 'Linda', 'linda@test.com');
+
+-- Get a reservation with any id, fx 1
+SELECT * FROM reservation WHERE id = 8;  
+
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE reservation 
+SET number_of_guests = 2 
+WHERE id = 8; 
+
+-- Delete a reservation with any id, fx 
+DELETE FROM reservation WHERE id = 8;
+ 
+-- Review Queries -- 
+-- Get all reviews
+SELECT * FROM review; 
+
+-- Add a new review
+INSERT INTO review(id, title, description, meal_id, stars, created_date) VALUES(9, 'Amazing Food', NULL , 3, 5, '2023-01-26');
+ 
+-- Get a review with any id, fx 1
+SELECT * FROM review WHERE id = 9;
+
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE review SET description = 'Super recommended!' WHERE id = 9;
+ 
+-- Delete a review with any id, fx 1
+DELETE FROM review WHERE id = 9; 
+
+--- additional queries -- 
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT meal.title, meal.when, meal.price 
+FROM meal 
+WHERE price <= 60; 
+
+-- Get meals that still has available reservations
+SELECT meal.title, (meal.max_reservation - SUM(number_of_guests)) AS available_reservations 
```

Comment:
`reservation.number_of_guests` would be nice to know what table it comes from, even though it is unnecessary.